fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -434,4 +473,16 @@ public boolean handleHbResponse(BackendHbResponse hbResponse) {
 
         return isChanged;
     }
+
+    public boolean isResourceAvailable() {
```

Comment:
Check the last report time, for the case BE may lost connect with FE and it would never report the resource usage.